R/get_publication.R

```diff
@@ -0,0 +1,97 @@
+#' Get publications
+#'
+#' @param page_size Number of results to collect in a single query to the API (max 40)
+#' @param page Page number to return (Default is NULL which will loop through until all
+#' pages of the query are collated).
+#' @param verbose Add extra contextual information whilst running
+#'
+#' @return Data frame listing all available publications
+#' @export
+#'
+#' @examples
+#' get_publication_catalogue()
```

Comment:
Also, while I'm on this particular train of thoughts (sorry!), for brevity, would `get_publications()` be an improvement on `get_publication_catalogue()`? Not sure the _catalogue adds much to the name